src/Molecules/Notification/Notification.test.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import { render } from '@testing-library/react'
+import '@testing-library/jest-dom/extend-expect'
+import userEvent from '@testing-library/user-event'
+import { ThemeProvider } from 'styled-components'
+
+import BaseNotification from './index'
+
+const Notification = props =>
+  <ThemeProvider theme={global.theme}>

```

Comment:
it could also be imported here from .storybook/themes, then no global settings are required